src/main/java/chess/dao/ChessGameDao.java

```diff
@@ -0,0 +1,89 @@
+package chess.dao;
+
+import chess.domain.piece.character.Team;
+import chess.exception.DataAccessException;
+import chess.exception.InvalidGameRoomException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class ChessGameDao {
+    private static final String TABLE_NAME = "chess_game";
+
+    public void add(Team currentTeam, String roomName, Connection connection) {
+        try {
+            validateRoomName(roomName, connection);
+            PreparedStatement statement = connection.prepareStatement(
+                    "INSERT INTO " + TABLE_NAME + "(room_name, current_team) VALUES (?, ?)");
```

Comment:
```suggestion
                    "INSERT INTO chess_game (room_name, current_team) VALUES (?, ?)");
```
굳이 chess_game을 빼서 가독성을 떨어뜨릴 필요가 있을까요?

Reply:
테이블 이름을 바꾸면서, 혹시나 놓친 부분이 생길까봐 상수처리했었습니다.

생각해보니, 테이블명이 자주 바뀔거같지도 않고 가독성만 해치는 느낌이 드네요. (실무에서 테이블명이 바뀌는 경우가 자주 있나요?)

수정했습니다!